packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -373,3 +374,58 @@ test.describe('`isDismissible` prop', () => {
         });
     });
 });
+
+test.describe('isOpen prop', () => {
+    test('should not render open when isOpen = false', async ({ mount, page }) => {
+        // Arrange
+        await mount(PieModal, {
+            props: {
+                isOpen: false,
+            },
+        });
+
+        // Assert
+        await expect(page.locator(modalSelector)).not.toBeVisible();
+    });
+
+    test('should render open when isOpen = true', async ({ mount, page }) => {
+        // Arrange
+        await mount(PieModal, {
+            props: {
+                isOpen: true,
+            },
+        });
+
+        // Assert
+        await expect(page.locator(modalSelector)).toBeVisible();
+    });
+});
+
+test('Should not be able to scroll when modal is open', async ({ page, mount }) => {
```

Comment:
in the previous tests we use the condition `when isOpen = true`. Should we stick to that for this test as well?